### ISSUE TITLE
[New build - DEFAULT] 24w29b: Hotfixes

### ISSUE THREAD

**Issue by tsunamayo** [tsunamayo/Starship-EVO#5885]
Some more hotfixes.
More crafting schematics have been added.

Hotfixes:
#5883 Warp pearl not loaded correctly
#5881 Cannot place capacitor
#5882 Item stand can causes collisions
#5876 Seat wrong position

**Comment by ProPeach** [comment]
Great to see the medbay able to be crafted!
Could you add the mining beam to the crafting menu too? It'll be great to get an upgrade over mining by hand

**Comment by tsunamayo** [comment]
@ProPeach Yes it is planned, although the schematic will be updated at one point.
